students/piechowski.michal/lesson_01_basics/apple_sharing.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
+
+numberOfStudents = int(input())
```

Comment:
Zapoznaj się z code convention w Pythonie. 
Informacje na ten temat opisuje tzw PEP8 (Style Guide for Python Code) i następnym razem staraj się do niego dostosować. Link jest w sekcji 'reference materials' w README.md z opisem kursu.